azure_pipelines/create_azure_build_agent_vm.yaml

```diff
@@ -0,0 +1,78 @@
+# This Azure DevOps pipeline can generate an Azure DevOps Agent image.
+#
+# It only runs on self hosted agents, including VMSS agents. It cannot be run on hosted agents, as the
+# runtime is longer than a self-hosted agent allows. This may create a chicken-egg problem, which can be resolved
```

Comment:
I don't think that we would like to provide this yaml in repository.
cc: @AlenaSviridenko do you think any thoughts about it?

Reply:
No problem; I found it convenient for a generic and maybe common use case: 
I only need a regular up-to-date build of the Azure DevOps Agent image. With this file being part of the repository, I could create a pipeline in my Azure DevOps organization, connected directly to this yaml file in the `actions/virtual-environments` repository. Scheduled builds could then potentially ensure we always have the latest images.

But I also see your point, that it probably does not fit perfectly into this repository. Also, it can be moved to a dedicated repo easily enough. I have removed it for now; in case you find it convenient later on, we can always add it back. 

Backup: https://gist.github.com/seqdan/92a8e84673ec6729bbd48e80bbbae125